Unix/tools/make_tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
might don't have bash on Unix.

Reply:
Need bash for other things. It already is a requirement.